include/rocksdb/db.h

```diff
@@ -705,33 +717,42 @@ class DB {
   // benefits.
   // Parameters -
   // options - ReadOptions
-  // column_family - ColumnFamilyHandle* that the keys belong to. All the keys
-  //                 passed to the API are restricted to a single column family
   // num_keys - Number of keys to lookup
+  // column_families - Pointer to C style array of ColumnFamilyHandle* that
+  //                   the keys belong to.
   // keys - Pointer to C style array of key Slices with num_keys elements
   // values - Pointer to C style array of PinnableSlices with num_keys elements
+  // timestamps - Pointer to C style of std::string that, if non-null and
+  //              timestamps are enabled, will have the timestamps of the keys
```

Comment:
Rule 0 of function documentation: what are the inputs and what are the outputs? It's not clear whether timestamps are inputs, outputs, or both. Probably not *just* input because not const, but not obvious.